packages/lpd8editor/PKGBUILD

```diff
@@ -0,0 +1,31 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Nils Hilbricht <nils -at- hilbricht -dot- net>
+
+_projectname=lpd8editor
+pkgname="${_projectname,,}"
```

Comment:
`${foo,,}` is just expanded to an all-lowercase version of the value of `foo`, so here you can just use:

```
pkgname=lpd8editor
```
and use `$pkgname` instead of `${_projectname}` in the rest of the file.